src/pyhf/infer/mle.py

```diff
@@ -38,15 +78,29 @@ def fit(data, pdf, init_pars=None, par_bounds=None, **kwargs):
 
 def fixed_poi_fit(poi_val, data, pdf, init_pars=None, par_bounds=None, **kwargs):
     """
-    Run a maximum likelihood fit with the POI value fixzed.
+    Run a maximum likelihood fit with the POI value fixed.
+
+    Example:
+        >>> import pyhf
+        >>> pyhf.set_backend("numpy")
+        >>> model = pyhf.simplemodels.hepdata_like(
+        ...     signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0]
+        ... )
+        >>> observations = [51, 48]
+        >>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
+        >>> test_poi = 1.0
+        >>> pyhf.infer.mle.fixed_poi_fit(test_poi, data, model, return_fitted_val=True)
+        (array([1.        , 0.97224597, 0.87553894]), 28.92218013492061)
 
     Args:
-        data: the data
+        data: The data
         pdf (~pyhf.pdf.Model): The statistical model adhering to the schema model.json
-        kwargs: keyword arguments passed through to the optimizer API
+        init_pars (`list`): Values to initialize the model paramters at for the fit
+        par_bounds (`list` of `list`\s or `tuple`\s): The extrema of values the model paramters are allowed to reach in the fit
+        kwargs: Keyword arguments passed through to the optimizer API
 
     Returns:
-        see optimizer API
+        See optimizer API
```

Comment:
add a link to optimizer API?

Reply:
Which API? We don't have a central optimizer API page and the different opitmizers mostly all just accept args of `**kwargs`. Example: [`optimize.opt_pytorch.pytorch_optimizer`](https://scikit-hep.org/pyhf/_generated/pyhf.optimize.opt_pytorch.pytorch_optimizer.html)